generate-spec

```diff
@@ -302,6 +302,12 @@ foreach ($parsedRoutes as $key => $value) {
 			continue;
 		}
 
+		$controllerScopes = Helpers::getAttributeScopes($controllerClass, 'OpenAPI', $routeName);
+		if (in_array('ignore', $controllerScopes, true)) {
+			Logger::info($routeName, "Controller '" . $controllerName . "' ignored because of OpenAPI attribute");
+			continue;
+		}
```

Comment:
If there's more than one scope but also the ignore scope, we should probably print an error because it was not intended.